frontend/src/app/libs/admin-role-management/components/users-role-management-panel/users-role-management-panel.component.ts

```diff
@@ -8,57 +8,150 @@ import { AuthFacadeService } from 'src/app/libs/auth/store/auth/auth.facade';
 import { EntityStatus } from 'src/app/libs/auth/models/entity-status';
 import { takeUntil } from 'rxjs/operators';
 import { UserSearchService } from '../../services/user-search.service';
+import { ColumnConfig, TableColumnType } from 'src/app/libs/common-components/components/table/models/row-config.enum';
+import { ComponentTheme } from 'src/app/libs/common-components/shared/component-theme.enum';
+import { RowToggleOutput } from 'src/app/libs/common-components/components/table/models/row-toggle-output.enum';
+import { TableComponent } from 'src/app/libs/common-components/components/table/table.component';
 
 @Component({
 	selector: 'app-users-role-management-panel',
 	templateUrl: './users-role-management-panel.component.html',
 	styleUrls: ['./users-role-management-panel.component.scss'],
 })
 export class UsersRoleManagementPanelComponent implements OnInit, OnDestroy {
-	public searchText: string = '';
-	public userTemplatesChanges$: Observable<SimpleChanges[]>;
+
 	public destroy$: Subject<boolean> = new Subject();
+
 	public userUpdate: EntityWrapper<User>;
+
 	public users: User[];
 
+	public isSelectedUsersRemoving: boolean;
+
+	public pendingStatus: string = EntityStatus.Pending;
+
+	public headerButtonText: string = 'delete';
+
+	public lightTheme: ComponentTheme = ComponentTheme.Light;
+	public darkTheme: ComponentTheme = ComponentTheme.Dark;
```

Comment:
It's possible to provide enum for view:
`public readonly componentTheme: typeof ComponentTheme = ComponentTheme;`

What do you think about that? It's not necessary changes, but if you agree with me, we can apply them.

Reply:
its a good idea and make our code shorter, I've made such changes to that component ans will apply this further. But previous pattern was as standard before and we have a lot of this implementations in our code.